app/views/user_mailer/_gpx_description.text.erb

```diff
@@ -0,0 +1,8 @@
+<% trace_name = @trace_name %>
+<% trace_description = @trace_description %>
+<% if @trace_tags.length > 0 %>
+  <% tags = safe_join @trace_tags.map { |trace_tag| trace_tag.tag }, ", " %>
+  <%= t ".description_with_tags_html", :trace_name => trace_name, :trace_description => trace_description, :tags => tags %>
```

Comment:
Tags are html-encoded: `&` to `&amp;`, `<` to `&lt;` etc.

Reply:
This should be fixed now. I also added some non _html translation strings, just in case. This is similar to the other user_mailer texts.